main.go

```diff
@@ -40,8 +46,8 @@ func main() {
 		}
 	})
 
-	router.LoadHTMLGlob("dist/*.html")
-	router.Static("/static", "./dist/static")
+	router.LoadHTMLGlob("ui/dist/*.html")
```

Comment:
is this correct? seems like you have it at a different path as https://github.com/jobsgowhere/jobsgowhere/pull/65/files#diff-b67911656ef5d18c4ae36cb6741b7965R61

Reply:
let me clean the cache and try again.

Reply:
This change is correct as the [above code](https://github.com/jobsgowhere/jobsgowhere/pull/65/files#diff-b67911656ef5d18c4ae36cb6741b7965R61) you pointed was added to make it work with Docker (but it still has some problem). I can comment that code to avoid confusion. 

`npm run build` creates the static files under `ui/dist` .... so we can proceed with 
`go run main.go` for now instead of `make run` 

I will take a look at what is wrong with Docker setup in another PR.